src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -554,7 +576,13 @@ private void displayItem(ChestMenu menu, PlayerProfile profile, Player p, Object
 
         for (int i = 0; i < 9; i++) {
             ItemStack recipeItem = getDisplayItem(p, isSlimefunRecipe, recipe[i]);
-            menu.addItem(recipeSlots[i], recipeItem, clickHandler);
+            MenuClickHandler handler;
+            if (recipeItem instanceof UnlockableItemStack) {
+                handler = lockedClickHandler;
+            } else {
+                handler = clickHandler;
+            }
```

Comment:
```suggestion
            MenuClickHandler handler = (recipeItem instanceof UnlockableItemStack) ? lockedClickHandler : clickHandler;

```
if statement works too but Id just stick to the ternary (though it is your choice)

Reply:
Walshy thought the if statement is better so

Reply:
no walshy just said to extract the ternary to a variable so the method call isnt a big mess
you can still use a ternary if pleased